EIPS/eip-5548.md

```diff
@@ -0,0 +1,121 @@
+description: An EIP-721 extension to deny specific smart contracts the ability to spend tokens on an owner's behalf
+author: Mitchell F Chan (@mitchellfchan), Harm van den Dorpel (@harmvandendorpel), Billy Rennekamp (@okwme)
+discussions-to: https://ethereum-magicians.org/t/eip-5548-eip-721-approve-operator-denylist/10549
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-27
+requires: 165, 721
+---
+
+## Abstract
+
+This EIP is an extension to [EIP-721](./eip-721.md) which standardizes NFT creators' ability to deny specific marketplaces or smart contracts the ability to spend tokens on an owner's behalf. This is achieved by adding a check against an owner-defined denyList in the `Approve` or `SetApprovalForAll` functions.
+
+This is intended for NFT creators who wish to exercise some control over where their creations may be bought and sold, without limiting the rights of token holders. 
+
+There are many reasons why an NFT creator may wish to exercise control over which intermediary smart-contracts may buy and sell their creations on an owner's behalf. An intermediary platform may present artwork in a manner which the creator finds objectionable or dishonest, or the intermediary platform may openly disregard off-chain codes, conventions, or practices which the creator considers to be an essential part of the artwork or their career.
+
+At the time of writing, several intermediary platforms have chosen to disregard the widely agreed-upon convention of remitting resale royalties to creators. It is expected that the extension proposed in this EIP will frequently be paired with [EIP-2981](./eip-2981.md), with EIP-2981 acting as a way of signalling standardized royalty information, and this EIP acting as a method of denying non-compliant intermediary platforms.
+
+
+## Motivation
+
+The promise of creator empowerment has driven adoption of the NFT standard (and of the Ethereum blockchain in general) for artists and creators. The opportunity to cut out intermediaries, or "middlemen," is a central tenet not just of NFT culture, but of crypto culture in general. Nonetheless, intermediaries persist. In many cases, these intermediaries serve an important function, creating smoother experiences for end users, or presenting NFT metadata in easily consumable formats. 
+
+However, as intermediary platforms willfully circumvent social codes established by NFT creators, it is within the spirit of crypto to provide technological tools to deny allowances to those intermediaries.
+
+It should be noted that this extension will not (nor does it aim to) stamp out any nefarious behaviour (such as royalty-dodging) from token holders. The position of this EIP is that any restriction to owner's ability to `Transfer` their NFTs would be overreaching. And of course, denyListed platforms could deploy new smart contracts, placing creators in the position of playing "whack-a-mole" to add new deployments to the denyList. 
```

Comment:
This paragraph belongs in Security Considerations, and should be expanded with more detail.